game_eggs/steamcmd_servers/stationeers/stationeers_bepinex/egg-stationeers--bep-in-ex.json

```diff
@@ -0,0 +1,154 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-10-28T19:39:03+02:00",
+    "name": "Stationeers (BepInEx)",
+    "author": "m1z@tuta.io",
```

Comment:
main egg is from me. SO dont change author. Respect MIT Licence

Reply:
Oh sorry about that. I'll fix that